package.json

```diff
@@ -38,18 +37,28 @@
     "body-parser": "1.15.2",
     "console": "^0.7.2",
     "dotenv": "4.0.0",
-    "eslint": "0.24.0",
     "express": "4.12.3",
     "pg-promise": "5.3.5",
     "promise": "7.1.1",
     "serverless": "1.9.0",
     "serverless-plugin-write-env-vars": "1.0.1",
+    "sinon": "^6.0.1",
     "swagger-express-mw": "0.6.0",
     "swagger-tools": "0.10.1"
   },
   "devDependencies": {
+    "assert": "^1.4.1",
```

Comment:
Can you remove the carets from the packages. We've found that at times a package may get a minor version bump which creates backwards compatibility issues.